src/stream/src/executor/aggregation/agg_group.rs

```diff
@@ -68,14 +68,7 @@ impl Strategy for AlwaysOutput {
                 // Generate output no matter whether current row count is 0 or not.
                 Some(RecordType::Insert)
             }
-            Some(prev_outputs) => {
-                if prev_row_count == 0 && curr_row_count == 0 || prev_outputs == curr_outputs {
-                    // No rows exist, or output is not changed.
-                    None
-                } else {
-                    Some(RecordType::Update)
-                }
-            }
+            Some(_prev_outputs) => Some(RecordType::Update),
```

Comment:
Better to add some comments here, since this is suboptimal in cases where no approx percentile is involved. Some one may want to change it back if no background is provided.

Also, I guess it can be reasonable to only do this when approx percentile appears. That means, we can have a new `Strategy` impl to instantiate another `AggGroup` and hence `SimpleAggExecutor`, which is only used in that specific case.

Reply:
That's my initial thought as well. But on discussion with @st1page and @fuyufjh, the overhead is minimal for simple agg, since it's just 1 record. Adding a new strategy will add more complexity to simple agg, since we need to specially dispatch the strategy, depending on whether approx percentile is present.

I will add some comments.

Reply:
@KeXiangWang